Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:3.8 as base
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python -
+ENV PATH="${PATH}:/root/.poetry/bin"
+
+WORKDIR /todo_app
+COPY pyproject.toml poetry.lock /todo_app/
+COPY ./todo_app /todo_app/todo_app
```

Comment:
You could do this COPY **after** the `poetry install` commands to speed up rebuilds - if you only change a .py file we don't want to run poetry install again on the rebuild.

Reply:
Done, thanks.